packages/financial-templates-lib/src/price-feed/MedianizerPriceFeed.js

```diff
@@ -49,6 +49,17 @@ class MedianizerPriceFeed extends PriceFeedInterface {
     }
   }
 
+  debugHistoricalData(time) {
+    let priceFeedErrorDetails = [];
+    this.priceFeeds.map(priceFeed => {
+      const hasPrice = priceFeed.getHistoricalPrice(time);
+      if (!hasPrice) {
+        priceFeedErrorDetails.push(`PriceFeed down with UUID: ${priceFeed.uuid}`);
```

Comment:
Might be useful to include the time that's unavailable in these since it may be particular times that are broken as opposed to the entire feed being down?